src/ctrl/nlas/ops.rs

```diff
@@ -4,13 +4,42 @@ use crate::constants::*;
 use anyhow::Context;
 use byteorder::{ByteOrder, NativeEndian};
 use netlink_packet_utils::{
-    nla::{Nla, NlaBuffer},
+    nla::{Nla, NlaBuffer, NlasIterator, NLA_F_NESTED},
     parsers::*,
     traits::*,
     DecodeError,
 };
 use std::mem::size_of_val;
 
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct Op {
+    pub nlas: Vec<OpAttrs>,
+}
+
+impl Nla for Op {
+    fn value_len(&self) -> usize {
+        self.nlas.iter().map(|nla| nla.buffer_len()).sum()
+    }
+
+    fn kind(&self) -> u16 {
+        NLA_F_NESTED
```

Comment:
This should be index of Op. This is kernel code:

```c
nla_ops = nla_nest_start_noflag(skb, CTRL_ATTR_OPS);
if (nla_ops == NULL)
	goto nla_put_failure;

while (genl_op_iter_next(&i)) {
	struct nlattr *nest;
	u32 op_flags;

	op_flags = i.flags;
	if (i.doit.policy || i.dumpit.policy)
		op_flags |= GENL_CMD_CAP_HASPOL;

	nest = nla_nest_start_noflag(skb, genl_op_iter_idx(&i));
```

Reply:
Fixed.